addons/web/static/src/core/notebook/notebook.scss

```
@@ -1,18 +1,76 @@
 .o_notebook {
-    &.horizontal {
-        flex-direction: column;
+    --notebook-margin-x: 0;
+    --notebook-padding-x: 0;
+    --notebook-link-border-color: transparent; 
+    --notebook-link-active-border-color: #{$border-color};
```

```suggestion
    --notebook-link-border-color-active: #{$border-color};
```
interaction states at the end of the variable name, the same for `hover` and `accent`;)